### PR TITLE
[Message Queue] Design Update

### PR DESCRIPTION
**Pull request by issacyxw**

### CODE REVIEW

**Comment by xieus** [comment]
@issacyxw You may need to update your master branch with the latest futurewei-cloud:master. This PR includes quite a few changes which have been merged to futurewei-cloud:master previously.

**Comment by xieus** [comment]
> @issacyxw You may need to update your master branch with the latest futurewei-cloud:master. This PR includes quite a few changes which have been merged to futurewei-cloud:master previously.

@issacyxw At the bottom of this page, you can see that there is quite a few of conflicting files with existing master branch. Can you fix the issue?

File: docs/modules/ROOT/pages/mq_services/message_queue_system.html [hunk 1]
```diff
@@ -526,16 +525,27 @@ <h3 id="_architecture">Architecture</h3>
...
+</div>
+</div>
 <div class="sect3">
 <h4 id="_design_option_mq_vpc_topic_grpc">Design Option: MQ (VPC Topic) + GRPC</h4>
 <div class="sect4">
-<h5 id="_workflow">Workflow</h5>
+<h5 id="_workflow_2">Workflow</h5>
 <div class="imageblock">
 <div class="content">
 <img src="../../images/vpcGroup.png" alt="vpcGroup">
```
Comment (xieus) [inline docs/modules/ROOT/pages/mq_services/message_queue_system.html:551]: Could you check the image name under /image folder? Should be vpcGroup.png, not vpcGrouppng. Thanks.

File: docs/modules/ROOT/workflows/MQ_portcreate.puml [hunk 2]
```diff
@@ -7,13 +7,23 @@ autonumber
 
 user -> PM: Create a port
 PM <-> SM_MACM_IPM_RM: PortInfo update
+<<<<<<< HEAD
```
Comment (xieus) [inline docs/modules/ROOT/workflows/MQ_portcreate.puml:10]: Line 10 is a merge conflict. Please review "<<<<<<<< HEAD"

File: docs/modules/ROOT/workflows/MQ_portcreate.puml [hunk 3]
```diff
@@ -7,13 +7,23 @@ autonumber
...
+DPM --> targetNodeACA: Query vpc_subs in Nodeinfo \n If target node doesn't subscribe topic, \n Using (MQ_url & unicast_topic & multicast_topic & vip_topic) in VpcInfo \n to call subscribe API with GRPC  \n Send the ack topic info (MQ_url & ack_topic) in VpcInfo
+=======
 PM <-> VPCM: VpcInfo query by vpc_vni
 PM <-> NM: Nodeinfo query by Node_id
 PM -> DPM: Send down network configuration request
 DPM ->DPM: Select chanel for this network configuration (MQ selected)\n Query vpc_subs in Nodeinfo \n subscribe the ack topic
 'DPM <-> VPCM: VPC info query
 'DPM <->NM: Query whether target node subscript vpc topic or not
 DPM --> targetNodeACA: CSA_Alg： Node subscribe API with GRPC\n Using (MQ_url & unicast_topic & multicast_topic & vip_topic) in VpcInfo \n Send the ack topic info (MQ_url & vip_topic) in VpcInfo
+>>>>>>> c7baf54731aabf851f79b69b96d539ab1eb3c24b
```
Comment (xieus) [inline docs/modules/ROOT/workflows/MQ_portcreate.puml:26]: Check the conflict block between Line 10-18 and Line 18-26. Please merge it properly.

File: docs/modules/ROOT/workflows/MQ_portcreate.puml [hunk 4]
```diff
@@ -31,13 +41,23 @@ autonumber
...
 PM <-> SM_MACM_IPM_RM: PortInfo update
+<<<<<<< HEAD
+PM <-> NM: Nodeinfo query by Node_id
+PM -> DPM: Send down network configuration request
+DPM <-> VPCM: VpcInfo query by vpc_vni
+DPM ->DPM: CSA-Alg: Select chanel for this network configuration (GRPC selected)
+'DPM <-> VPCM: VPC info query
+'DPM <->NM: Query whether target node subscript vpc topic or not
+DPM --> targetNodeACA: call node unsubscribe API with GRPC\n Using (MQ_url & unicast_topic & multicast_topic & vip_topic) in VpcInfo
+=======
```
Comment (xieus) [inline docs/modules/ROOT/workflows/MQ_portcreate.puml:52]: Same here. Merge conflict.

File: docs/modules/ROOT/pages/mq_services/message_queue_system.html (hunk 1 above)
Comment (xieus) [reply docs/modules/ROOT/pages/mq_services/message_queue_system.html:551]: @issacyxw Where is the image vpcGroup.png?

**Review by xieus — CHANGES_REQUESTED** [review]
There are some merge conflict we have to resolve before check in to master.

**Comment by xieus** [comment]
@issacyxw Please edit the title and add more details to the PR description.

File: docs/modules/ROOT/workflows/MQ_portcreate.puml (hunk 2 above)
Comment (issacyxw) [reply docs/modules/ROOT/workflows/MQ_portcreate.puml:10]: I have fixed MQ_portcreate.puml

File: docs/modules/ROOT/pages/mq_services/message_queue_system.html (hunk 1 above)
Comment (issacyxw) [reply docs/modules/ROOT/pages/mq_services/message_queue_system.html:551]: vpcGroup.png exists in the  /image folder

**Review by xieus — APPROVED** [review]
Thanks for the continued improvement.

**Pull request merged**